### PR TITLE
added loading skeleton for page

### LINKED ISSUE #25: add loading skeletons
make it so when we are loading we have some skeletons that animate to take up the space while waiting for data.

can be in a `loading.tsx` file that handles it in next

### PR DESCRIPTION
**Pull request by JoshHyde9**
Fixes #25

### CODE REVIEW

**Review by RossLitzenberger — APPROVED** [review]

File: src/app/loading.tsx [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+    <div className="flex min-h-screen flex-col p-4 md:p-24">
+      <div className="h-10 animate-pulse bg-neutral-700 rounded-full dark:bg-gray-700 ml-8 mb-6 md:mb-20 w-4/5"></div>
+      <div className="flex flex-col items-center animate-pulse">
+        <div className="flex flex-col justify-start gap-2 w-full mt-4 mb-8">
+          <div className="flex flex-col justify-start gap-2 w-full mb-8 ml-8">
+            <div className="h-6 bg-neutral-700 rounded-full dark:bg-gray-700 w-3/5 mb-4"></div>
+            <div className="h-3 bg-neutral-700 rounded-full dark:bg-gray-700 w-2/5 mb-4"></div>
+          </div>
+
+          {new Array(5).fill("").map((_, i) => (
```
Comment (jhechtf) [inline src/app/loading.tsx:14]: I'd personally pull this up into the function itself and set it like so 
```ts
const placeholders = Array.from({ length: 5 }, () => '');
///
return (
///
{placeholders.map(...)
);
```

Comment (JoshHyde9) [reply src/app/loading.tsx:14]: What is the benefit of this? Re-usability?  Or am I a dumbass?

**Comment by Hacksore** [comment]
I just realized we might never see this loading state if we ISR as the server will always give you HTML with data 😂 

It’s still ok to have tho 🫡

**Comment by JoshHyde9** [comment]
> I just realized we might never see this loading state if we ISR as the server will always give you HTML with data 😂
> 
> It’s still ok to have tho 🫡

How did I not think of this

**Comment by Hacksore** [comment]
It’s possible we still see it when the server is doing a revailidate methinks

**Pull request merged**